lib/yargs-factory.ts

```diff
@@ -655,7 +655,7 @@ function Yargs(
     return self;
   };
 
-  self.command = function (
+  self.command = self.commands = function (
```

Comment:
This seems unrelated to the other work? Could this be a separate PR?

Reply:
Done